### PR TITLE
Added handling of the mouse side button function

### PR DESCRIPTION
**Pull request by zgml**
Rabbit's rdp connection does not handle the mouse side button function, resulting in a remote crash if the user presses the mouse side button while remote

### CODE REVIEW

**Comment by KangLin** [comment]
Please See:
https://github.com/KangLin/RabbitRemoteControl/blob/d5514076a823d0bcb218c38a682e5c92c46a9b60/Client/ConnectDesktop.h#L102
to
https://github.com/KangLin/RabbitRemoteControl/blob/d5514076a823d0bcb218c38a682e5c92c46a9b60/Client/ConnectDesktop.h#L117

please modify 
```
virtual void mousePressEvent(QMouseEvent *event);
virtual void mouseReleaseEvent(QMouseEvent *event);
virtual void mouseMoveEvent(QMouseEvent *event);
virtual void wheelEvent(QWheelEvent *event);
virtual void keyPressEvent(QKeyEvent *event);
virtual void keyReleaseEvent(QKeyEvent *event);
```

File: Plugins/FreeRDP/Client/ConnectFreeRDP.cpp [hunk 1]
```diff
@@ -1720,45 +1724,81 @@ void CConnectFreeRDP::mouseMoveEvent(QMouseEvent *event)
...
+    if (flags != 0)
     {
-        flags |= PTR_FLAGS_BUTTON3;
+        SendMouseEvent(flags, pos, isExtended);
     }
-    SendMouseEvent(flags, event->pos());
 }
 
-void CConnectFreeRDP::mouseReleaseEvent(QMouseEvent *event)
+void CConnectFreeRDP::slotMouseReleaseEvent(Qt::MouseButton button, QPoint pos)
```
Comment (KangLin) [inline Plugins/FreeRDP/Client/ConnectFreeRDP.cpp:1767]: Please use:
    virtual void mouseReleaseEvent(QMouseEvent *event)

Comment (zgml) [reply Plugins/FreeRDP/Client/ConnectFreeRDP.cpp:1767]: My error, the code I used for debugging is old

**Comment by KangLin** [comment]
Add you to Contributors in Authors.md and Authors_zh_CN.md .

**Comment by KangLin** [comment]
@zgml please modify build error. then `git rebase`

**Comment by zgml** [comment]
I think I submitted it right this time

Kang Lin ***@***.***> 于2024年12月2日周一 04:32写道：

**Comment by KangLin** [comment]
There are also two formatting issues. After modification, it can be merged.

**Comment by KangLin** [comment]
Also, you need to use `git rebase`  to clean up the commit history. Turn it into a commit record.

**Pull request closed without merging**